loki/transform/transform_hoist_variables.py

```diff
@@ -131,7 +132,16 @@ def transform_subroutine(self, routine, **kwargs):
 
         role = kwargs.get('role', None)
         item = kwargs.get('item', None)
-        successors = kwargs.get('successors', None)
+        _successors = kwargs.get('successors', None)
+        successors = []
+        for successor in _successors:
+            append = True
+            for _disable in self.disable:
```

Comment:
I think the disable functionality isn't covered by any of the tests? (suggests [Codecov](https://app.codecov.io/gh/ecmwf-ifs/loki/commit/1949d06af2103e5679dd2c5de724b182747b6ddb/loki/transform/transform_hoist_variables.py))

Reply:
Yes ... it is "tested" within the CLOUDSC SCC-CUF transformation. 
I'll add a test.

Reply:
Right, thanks! In that case it's not high-priority although a test directly in the test base is of course appreciated. The CLOUDSC regression tests are not included in the coverage